release.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+RELEASE_LEVEL=$1
+
+git remote -v | grep "^upstream.*github\.com.jfly\/shtuff\.git" &> /dev/null
+
+if [[ $? -gt 0 ]]; then
+    echo "A remote named upstream must be configured to point to https://github.com/jfly/shtuff.git to be able to make a release"
+    exit 1
+fi
+
+git fetch upstream &> /dev/null
+
+if [[ $? -gt 0 ]]; then
+    echo "Fetch did not perform successfully, make sure you hav permission to pull from this repository."
+    exit 2
+fi
+
+CURRENT_VERSION=$(git tag | sort -V | tail -n 1)
+NEW_VERSION=$(python -c "from bump_version import bump_version; print(bump_version('$CURRENT_VERSION', '$RELEASE_LEVEL'))")
```

Comment:
I think it would be cleaner to actually define a main function in bump_version. Then you could change this to something like:

```
NEW_VERSION=$(./bump_version $CURRENT_VERSION $RELEASE_LEVEL)
```